ansible_image_validation/validation-playbooks/nvme_driver_validation.yml

```diff
@@ -0,0 +1,14 @@
+###
+# This Playbook checks if the nvme and pci drivers are present in the given image
+# Author: Navpreet Kaur
+###
+---
+- name: Check if the nvme and pci drivers are present i all images
+  shell: lsinitrd /boot | grep "{{item}}"
+  register: check_drivers_allimages
+  with_items:
+    - nvme
+    - pci 
+
+- debug:
+    var: check_drivers_allimages    
```

Comment:
add a task where you check whether the nvme files are present in initramfs or not